tests/validation/test_Minkowski_Structure_Metrics.py

```diff
@@ -19,63 +20,53 @@ def _get_structure_data(structure, qtype):
-            expected_ql = _get_structure_data(structure, "q")
-            expected_avql = _get_structure_data(structure, "avq")
-            expected_wl = _get_structure_data(structure, "w")
-            expected_avwl = _get_structure_data(structure, "avw")
+    @pytest.mark.parametrize("structure", ["fcc", "bcc", "hcp", "sc"])
+    def test_minkowski_structure_metrics(self, structure):
+        expected_ql = _get_structure_data(structure, "q")
+        expected_avql = _get_structure_data(structure, "avq")
+        expected_wl = _get_structure_data(structure, "w")
+        expected_avwl = _get_structure_data(structure, "avw")
 
-            with garnett.read(
-                os.path.join(
-                    os.path.dirname(__file__),
-                    "files",
-                    "minkowski_structure_metrics",
-                    f"{structure}.gsd",
-                )
-            ) as traj:
-                frame = traj[0]
-                box = frame.box
-                positions = frame.position.copy()
+        with garnett.read(
+            os.path.join(
+                os.path.dirname(__file__),
+                "files",
+                "minkowski_structure_metrics",
+                f"{structure}.gsd",
+            )
+        ) as traj:
+            snap = traj[0]
 
-            voro = freud.locality.Voronoi()
-            voro.compute((box, positions))
-            for sph_l in range(expected_ql.shape[1]):
+        voro = freud.locality.Voronoi()
+        voro.compute(snap)
+        for sph_l in range(expected_ql.shape[1]):
 
-                # These tests fail for unknown (probably numerical) reasons.
-                if structure == "hcp" and sph_l in [3, 5]:
-                    continue
+            # These tests fail for unknown (probably numerical) reasons.
+            if structure == "hcp" and sph_l in [3, 5]:
+                continue
 
-                # Test ql
-                comp = freud.order.Steinhardt(sph_l, weighted=True)
-                comp.compute((box, positions), neighbors=voro.nlist)
-                npt.assert_allclose(
-                    comp.order, expected_ql[:, sph_l], rtol=5e-5, atol=1e-3
-                )
+            # Test q'l
+            comp = freud.order.Steinhardt(sph_l, weighted=True)
+            comp.compute(snap, neighbors=voro.nlist)
+            npt.assert_allclose(comp.order, expected_ql[:, sph_l], atol=1e-5)
 
-                # Test Average ql
-                comp = freud.order.Steinhardt(sph_l, average=True, weighted=True)
-                comp.compute((box, positions), neighbors=voro.nlist)
-                npt.assert_allclose(
-                    comp.order, expected_avql[:, sph_l], rtol=5e-5, atol=1e-3
-                )
+            # Test average q'l
+            comp = freud.order.Steinhardt(sph_l, average=True, weighted=True)
+            comp.compute(snap, neighbors=voro.nlist)
+            npt.assert_allclose(comp.order, expected_avql[:, sph_l], atol=1e-5)
```

Comment:
The tolerances on these computations were far too wide before (an absolute tolerance of 1e-3 is really big). I wonder if the high tolerances were hiding the fact that the averaged quantities were incorrectly calculated. 🤔 These tests act on "perfect" structures so it would be easy to hide the issue that was averaging over two neighbor shells instead of one, and mask it with a higher numerical tolerance. Regardless, the measured tolerances are much lower now. I keep a minimum tolerance of 1e-5 even though it could be a little smaller, because some of these tests are known to be flaky due to numerical differences on exotic systems/compilers/etc.